### PR TITLE
Connection-token-handler component source code

### PR DESCRIPTION
**Pull request by crabtree**
**Description**

Changes proposed in this pull request:

- Controller for connector service token requests implementation

**Related issue(s)**

See also #1152

### CODE REVIEW

File: components/connector-service-token-request/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ARG DOCK_PKG_DIR=/go/src/github.com/kyma-project/kyma/components/connector-service-token-request
+
+RUN mkdir -p $DOCK_PKG_DIR
+
+COPY ./ $DOCK_PKG_DIR
+WORKDIR $DOCK_PKG_DIR
+
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o manager ./cmd/manager
+
+FROM alpine
```
Comment (a-thaler) [inline components/connector-service-token-request/Dockerfile]: we should use fixed versions. In my understanding that image is like "latest"

Comment (crabtree) [reply components/connector-service-token-request/Dockerfile]: Good catch! I missed that. Fixed.

File: components/connection-token-handler/cmd/manager/manager.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	if err != nil {
+		log.Error(err, "unable to set up overall controller manager")
+		os.Exit(1)
+	}
+
+	log.Info("Registering Components.")
+
+	log.Info("setting up scheme")
+	if err := apis.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "unable add APIs to scheme")
```
Comment (janmedrek) [inline components/connection-token-handler/cmd/manager/manager.go]:
```suggestion
		log.Error(err, "unable to add APIs to scheme")
```

File: components/connection-token-handler/sample-cr.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (janmedrek) [inline components/connection-token-handler/sample-cr.yaml]: Do we need this sample? If yes - shouldn't it be placed somewhere near the CR and not inside controller's code?

Comment (crabtree) [reply components/connection-token-handler/sample-cr.yaml]: No we don't, removed :)

**Pull request merged**